Add Vishal to the contributors list

Mvishal123 opened the pull request.

Commit:
Add Vishal to the contributors list

Commit:
Added my name to Contributors.md file

iArmanKarimi reviewed (changes requested):
You've already added your name

Mvishal123 committed:
Removed the duplicate

- I had added my name twice by mistake. Now I have removed one of it.
- I have also removed few blank lines

Mvishal123 commented:
Sorry. I thought it wasn't merged

iArmanKarimi commented:
It hasn't been merged yet, I'm stating the duplicate.

Mvishal123 commented:
Hi I have removed the duplicate

Roshanjossey reviewed (commented):
Hello @Mvishal123, congratulations! You've successfully submitted a pull request. 🎉

  If you liked the tutorial, please star this repo by clicking star button on top right of this page.
  <img alt="star screenshot" title="star button" src="https://firstcontributions.github.io/assets/star.png">


  # Next steps
  - Continue contributing: If you're looking for projects to contribute to, checkout our [<img src="https://avatars0.githubusercontent.com/u/65761570?s=88&u=640f39b808c75c6b86460aa907dd030bcca2f3c7&v=4" width="22" title="web app" /> webapp](https://firstcontributions.github.io).
  - Join our slack group: We have a community to help/support contributors. [<img src="https://edent.github.io/SuperTinyIcons/images/svg/slack.svg" width="22" title="Slack" /> Join slack group](https://join.slack.com/t/firstcontributors/shared_invite/zt-vchl8cde-S0KstI_jyCcGEEj7rSTQiA).
  - Share on social media: You can share this content to help more people. [ <img alt="twitter" title="twitter" src="https://edent.github.io/SuperTinyIcons/images/svg/twitter.svg" width="22"> tweet](https://twitter.com/intent/tweet?text=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20@1stcontribution.%20You%20can%20too%20at%20https%3A//goo.gl/66Axwe%0A&hashtags=OpenSource,CodeNewbie). [<img alt="twitter" title="twitter" src="https://edent.github.io/SuperTinyIcons/images/svg/facebook.svg" width="22"> share](https://www.facebook.com/sharer/sharer.php?u=https://roshanjossey.github.io/first-contributions&quote=Yay%21%20I%20just%20made%20my%20first%20open%20source%20contribution%20with%20First%20Contributions.%20You%20can%20too,%20by%20following%20a%20simple%20tutorial%20at%20https%3A//goo.gl/66Axwe&hashtag=%23OpenSource)

We'd love to hear your thoughts about this project. Let us know how we can improve by commenting or opening an issue here.

![celebration gif](https://media2.giphy.com/media/26gN16cJ6gy4LzZSw/giphy.gif)
